pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -190,12 +183,13 @@ sub _suggest_imunify360 {
             block_notify => 1,                                                                                                      # Do not send a notification about this
         );
     }
-    elsif ( $licensed && !$installed ) {
+    elsif ( !Whostmgr::Imunify360::is_imunify360_installed() ) {
```

Comment:
Should we also check if is licensed here?

Reply:
It's not logically possible to reach this block in an unlicensed state because of the `if` and `elsif` above it.

Reply:
Nevermind, I guess this is not needed, because of the previous check